ex04.ex

```diff
@@ -55,7 +55,7 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(x), do: reduce(x, &(min(&1, &2)))
```

Comment:
## -4

`min` is a built-in library function, which is rubric says is not allowed.